dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/ContentsWebAPI.java

```diff
@@ -620,37 +620,14 @@ public Relationship getRelationshipByName(String relationshipName) {
 
 	public List<Relationship> getRelationshipsOfContentlet(String contentletInode, boolean hasParent) throws DotDataException, DotSecurityException {
 		Contentlet cont = conAPI.find(contentletInode, user, true);
-		return FactoryLocator.getRelationshipFactory().byContentType(cont.getStructure(), hasParent);
-	}
-
-	/**
-	 * This gets the list of all the relationship objects associated to the
-	 * structure of the contentlet
-	 *
-	 * @param cont
-	 *            The contentlet
-	 * @param hasParent
-	 *            true If you find the relations where the contentlet is parent,
-	 *            false If you find the relations where the contentlet is child
-	 * @return A list of relationship objects
-	 * @deprecated this methods could be found on StructuresWebAPI
-	 */
-	@Deprecated
-	public List<Relationship> getRelationshipsOfStructure(Structure st, boolean hasParent) {
-		return getRelationshipsOfStructure(st.getInode(), hasParent);
-	}
-	@Deprecated
-	public List<Relationship> getRelationshipsOfStructure(long structureInode, boolean hasParent) {
-		return getRelationshipsOfStructure(((Long) structureInode).toString(), hasParent);
-	}
-	@Deprecated
-	public List<Relationship> getRelationshipsOfStructure(String structureInode, boolean hasParent) {
-		Structure st = (Structure) InodeFactory.getInode(structureInode, Structure.class);
-		return FactoryLocator.getRelationshipFactory().byContentType(st, hasParent);
+		if(hasParent){
```

Comment:
it could be
```
hasParent? FactoryLocator.getRelationshipFactory().byParent(cont.getContentType()):
FactoryLocator.getRelationshipFactory().byChild(cont.getContentType());
```